### PR TITLE
Set prod flag correctly in hack/delete.sh 

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

File: test/manifests/normal/scaleup.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+location: "{{ .Env.AZURE_REGION }}"
```
Comment (kargakis) [inline test/manifests/normal/scaleup.yaml]: https://github.com/openshift/openshift-azure/issues/879

Comment (charlesakalugwu) [reply test/manifests/normal/scaleup.yaml]: just ran into this :)

Comment (kargakis) [reply test/manifests/normal/scaleup.yaml]: Note that scale up with a partial request is still not going to work in prod, see https://github.com/openshift/openshift-azure/issues/876.

**Pull request merged**